tests/testing_helpers/db.go

```diff
@@ -0,0 +1,18 @@
+package test_helpers
```

Comment:
Should this be `package testing_helpers` to make it consistent with directory name?